app/src/main/java/com/infomaniak/mail/ui/newMessage/AiViewModel.kt

```diff
@@ -79,10 +78,15 @@ class AiViewModel @Inject constructor(
         )
 
         ensureActive()
-        handleAiResult(apiResponse, userMessage)
+        val isUsingPreviousMessageAsContext = previousMessageBodyPlainText != null
```

Comment:
I'm not sure this var is needed
I would rename the handleApiResult's argument to 'isUsingPreviousMessageAsContext', because it would make sense in :
 ```
if(isUsingPreviousMessageAsContext) {
     discardPreviousMailContext()
     CONTEXT_TOO_LONG
} else {
      PROMPT_TOO_LONG
 }
```

Then the var would be useless

Reply:
Yeah, I agree that the naming is better this way which means that I don't need the intermediate variable anymore